test-data/apply-configuration/eventing-01/expected-output/Management/Create/namespaces/openshift-example-operator/core/events/004-body-example-operator.18073f8ffae5c728.yaml

```diff
@@ -1,17 +1,17 @@
 apiVersion: v1
 count: 1
 eventTime: null
-firstTimestamp: "2024-10-25T20:35:38Z"
+firstTimestamp: "2024-11-12T14:41:09Z"
```

Comment:
Questionable. Is the fake clock wiring broken?

Reply:
It was a good exercise for me.

The event recorder doesn't accept a clock and uses [~now](https://github.com/openshift/library-go/blob/master/pkg/operator/events/recorder.go#L222) when generating events.
In the future we could actually inject a clock to the event recorder.

But the equivalence function ignores events: 
https://github.com/openshift/multi-operator-manager/blob/afa4d013fe6f04859e1e477edd889f1072ca77bf/pkg/library/libraryapplyconfiguration/equivalence.go#L14

Thus, updating events should not be necessary.

Reply:
done.

Reply:
Simply removing it from the diff isn't the point. Let's get a PR merged into library go to take a passive clock

Reply:
This is how it works today. The integration tests don't produce a diff because the events are ignored. This PR does not change this behavior, so there is no need to block it.